homework-projects/guides/demo-day-presentation.md

```diff
@@ -0,0 +1,36 @@
+# Demo Day Presentation
+
+Your team may choose to run a demo day to present your project and get feedback. If so, here's a brief to guide you through your 15-minute presentation. Each individual will have 40-minutes in total, broken down into 15-minutes for presenting and 25-minutes for discussions. Your presentation will be an opportunity to showcase your project, detail your weekly progress, and share your challenges and learnings.
+
+
+## Presentation Structure (15 minutes)
+
+### Introduction (2 minutes)
+Briefly introduce your chosen project. (Refer to the "projects" folder for details.)
+
+### Weekly Task Breakdown (10 minutes)
+
+- JS2 Week 1: Discuss how you integrated Browser environment, DOM manipulation, and DOM event listeners into your project.
+- JS2 Week 2: Describe your use of Array functions and Arrow functions.
+- JS2 Week 3: Explain your implementation of Callback functions, Asynchronicity, and Scope.
+- JS3 Week 1: Share how you worked with JSON, APIs, and Fetch.
+- JS3 Week 2: Talk about your application of Promises and Async/Await.
+
+For each week, highlight:
```

Comment:
[suggestion] Not a native English speaker, but somehow it sounds better to me if it's inversed:
```suggestion
Highlight for each week:
```

Reply:
I think the original sounds better. English is my first language, but it should sound good to everyone! I'll think if there's a different way to phrase it :)